migrations/migrations/20201221141905-create-user.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) =>
+    queryInterface.createTable('Users', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      firstName: {
```

Comment:
To respect the database conventions we decided to keep tables and columns in snake_case.

show 3.3- Database: 
https://github.com/Wolox/tech-guides/blob/master/nodejs/docs/node-standard-and-best-practices.md